themes/bootstrap3/templates/Recommend/VisualFacets.phtml

```diff
@@ -31,7 +31,7 @@
 ?>
 
 <?php if (isset($visualFacetSet)): ?>
-  <script type="text/javascript">
+  <script type="text/javascript" nonce="<?=$this->cspnonce()?>">
```

Comment:
Would it be better to use the inlineScript helper here?

Reply:
definitaly, I've done it

Reply:
as it was last occurence of using CspNonce view helper, I even removed that helper